gluten-core/src/main/scala/io/glutenproject/execution/BasicScanExecTransformer.scala

```diff
@@ -33,7 +33,7 @@ import org.apache.spark.sql.execution.InSubqueryExec
 import org.apache.spark.sql.types._
 import org.apache.spark.sql.vectorized.ColumnarBatch
 
-trait BasicScanExecTransformer extends TransformSupport with GlutenPlan {
+trait BasicScanExecTransformer extends TransformSupport {
```

Comment:
Why remove the depends of `GlutenPlan`? I think it's not proper even nothing important implemented in it. This is a `trait` that all Gluten plan node inherited from, no matter it supports transform or not.